docs/content/docs/ops/state/checkpoint_vs_savepoint.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Checkpoint VS Savepoint"
```

Comment:
```suggestion
title: "Checkpoint vs. savepoints"
```

Reply:
I don't really see the difference between `VS` and `vs.` according to grammatic `VS`, `or.`, `vs` and `v.` are equal. I like `VS` more since it looks more explicit in the title